plutus-core/plutus-ir/test/lets/letInLet.golden

```diff
@@ -1,9 +1,3 @@
 (program 1.0.0
-  [
```

Comment:
Not sure why this didn't get removed before. I guess we're just running dead code more often or something?

Reply:
Yes, probably due to:
- more runs of the simplifier,
- inlining type abstractions.

Notice that LHS of the diff is after running the simplifier once, so there might have been some constructs that were removed already.